util/icon/extract_timings.py

```diff
@@ -160,6 +169,9 @@ def parse_time(time_string):
         h = 0
         m = 0
     else:
+        s = 0
+        m = 0
+        h = 0
         logger.error("did not match regex")
```

Comment:
So far `logger.error` was always followed by `sys.exit(1)`.
Convert to logger.warning instead?

Reply:
I agree that makes sense.